spec/integration/recipe_show_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe 'Public Recipe show', type: :system do
+  describe 'index page' do
+    before(:example) do
+      @user = User.create(
+        name: 'Gabriel',
+        email: 'didier@gmail.com',
+        password: 'anything',
+        confirmed_at: Time.now.utc
+      )
+
+      # sign_in @user
+      @recipe = Recipe.create(
```

Comment:
- [ ] It's a great implementation in capybara testing. However, we also need unit tests for this project as the previous reviewer mentioned. Please add unit tests to your project.

Reply:
Thanks @Rustamxon7 for the suggestion. They should add this on the requirements. Already added validations tests.